contracts/externals/openzeppelin-solidity/contracts/examples/SampleCrowdsale.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.24;
+pragma solidity ^0.5.0;
```

Comment:
Including this line, some lines are updated to `0.5.0`, not to `0.5.6`. Is it intended?

Reply:
Yes. All openzeppelin codes are used 0.5.0 not specific 0.5.x.
But in our code (bridge, token), I used 0.5.6 version.